.storybook/custom-decorators/toast.decorator.js

```diff
@@ -1,18 +1,25 @@
 import { useEffect } from "@storybook/client-api";
 import { showToast } from "../../src/toasts/toasts.js";
 
-function getButton() {
-  return document.querySelectorAll(".toast-clicked");
-}
+let count = 0;
 
 export const withToast = storyFn => {
   useEffect(() => {
-    getButton().forEach(tab =>
-      tab.addEventListener(
-        "click",
-        showToast({ description: "This is a toast" })
```

Comment:
This was a bug -- the second argument to `addEventListener` should be a function. However, we were calling showToast immediately instead of passing in a function that calls showToast.